Makefile

```diff
@@ -119,7 +120,7 @@ else
 
 $(SYMTABSRC): $(foreach SDIR, $(CONFIGURED_APPS), $(SDIR)_all)
 	$(Q) $(MAKE) install
-	$(Q) $(APPDIR)$(DELIM)tools$(DELIM)mksymtab.sh $(BINDIR) >$@.tmp
+	$(Q) $(APPDIR)$(DELIM)tools$(DELIM)mksymtab.sh $(BINDIR) $(SYMTABADDS) >$@.tmp
```

Comment:
let's append `-a` to `$(SYMTABADDS)`

Reply:
> let's append `-a` to `$(SYMTABADDS)`

This makes the code easier to understand, but it will "not" support multiple "-a" options like:
```Makefile
SYMTABADDS="-a $(XXXX_PREFIX)/platform/nuttx/exports_xxxx_api.txt"
ifneq ($(CONFIG_LIBM),)
    SYMTABADDS+="-a $(XXXX_PREFIX)/platform/nuttx/exports_xxxx_api_math.txt"
endif
```

The name `SYMTABADDS` is not really appropriate, it actually means an extra parameter.
Should we use more general name, e.g. "SYMTABEXT"?

Reply:
but SYMTABLST already indicate it's a list of directories and files

Reply:
> but SYMTABLST already indicate it's a list of directories and files

How about renaming to "SYMTABEXT"? (means EXTRA parameters for mksymtab.sh)

Reply:
Done.